sdks/java/io/cdc/src/main/java/org/apache/beam/io/cdc/DebeziumOffsetTracker.java

```diff
@@ -25,28 +25,46 @@
 
 import java.util.Map;
 
+/**
+ * {@link RestrictionTracker} for Debezium connectors
+ */
 public class DebeziumOffsetTracker extends RestrictionTracker<DebeziumOffsetHolder, Map<String, Object>> {
     private static final Logger LOG = LoggerFactory.getLogger(DebeziumOffsetTracker.class);
 
     private DebeziumOffsetHolder restriction;
     private static final long MILLIS = 60 * 1000;
+    private final Integer maxRecords = 10;
 
     DebeziumOffsetTracker(DebeziumOffsetHolder holder) {
         this.restriction = holder;
     }
 
+    /**
+     * Overriding {@link #tryClaim} in order to stop fetching records from the database.
+     *
+     * <p>Currently this works time-based:</p>
```

Comment:
You need to update this. Now it supports both approaches (time and count) based. 

Which approach has precedence over the other one and why? Seems like the time-based has precedence over count-based, we need to clarify this in the documentation.

Reply:
Done!